packages/vue/src/components/authenticator.vue

```diff
@@ -105,7 +106,7 @@ onMounted(() => {
 });
 
 onUnmounted(() => {
-  if (unsubscribeMachine) unsubscribeMachine();
```

Comment:
This `if` statement is unnecessary, because `unsubscribeMachine` is definitely assigned on init: 

https://github.com/aws-amplify/amplify-ui/blob/a419f3c9e1bfd1f57fbd318a76d22747e6190cb5/packages/vue/src/components/authenticator.vue#L73